about and chart components

shiyu3169 opened the pull request.

zachferland committed:
Merge pull request #12 from zachferland/add-reducer-tests

add minimal reducer tests

Commit:
about and chart components

brandenrodgers commented on app/components/Chart/ProfileChart.js:
```diff
@@ -0,0 +1,145 @@
+import {Link} from 'react-router'
+
+var LineChart = require("react-chartjs").Line;
+var PieChart = require("react-chartjs").Pie;
```
When you install these, make sure you add "--save" at the end of your command. That way it gets added to the packages.json

brandenrodgers commented on app/views/profile/ProfileView.js:
```diff
@@ -2,16 +2,17 @@ import React, { Component } from 'react';
-        <h2> This is the profile page</h2>
-          This is an example of using a component
-        <Hello />
-        All users found from http request
```
This looks good. You might run into some trouble with Shivy's work on this. I think she edited the same lines so you guys will have to figure out how you want it to look.

Commit:
Merge remote-tracking branch 'refs/remotes/CS4500/master' into add-component-chart-about

Commit:
changes to profile page.

1. Applied react-bootstrap on profile page and its components
2. Added featured events components and linked in profile page
3. Added popup feature when editing user information
4. Added upload photo button (doesn't function yet)
5. Organized profile page and its components
